### PR TITLE
Fix Wifi stress test crash issues

### PR DESCRIPTION
**Pull request by kishore-sn**
Two crash issues are found during wifi stress test and fixed in this PR. 

1. Usage fault (INVPC) while running any stress test
2. Usage fault (INVSTATE) or busfault while running "stress 1" command multiple times.

### CODE REVIEW

**Review by jeongchanKim — APPROVED** [review]
LGTM

**Review by deepaksrma — APPROVED** [review]
LGTM

**Review by thapav — APPROVED** [review]
LGTM

File: os/arch/arm/src/armv7-m/up_schedulesigaction.c [hunk 1]
```diff
@@ -186,7 +186,6 @@ void up_schedule_sigaction(struct tcb_s *tcb, sig_deliver_t sigdeliver)
 #else
 				current_regs[REG_PRIMASK] = 1;
 #endif
-				current_regs[REG_XPSR] = ARMV7M_XPSR_T;
```
Comment (sangam-swami) [inline os/arch/arm/src/armv7-m/up_schedulesigaction.c:189]: are these changes tested with kernel signal tc's and others?

Comment (kishore-sn) [reply os/arch/arm/src/armv7-m/up_schedulesigaction.c:189]: Yes. Kernel tc have been checked.

**Comment by sunghan-chang** [comment]
@kishore-sn Let me ask the person who tested to test again with this PR.

**Review by davidfather — APPROVED** [review]
Looks good to me.

**Comment by vibhor-m** [comment]
Hello All,

My testing environemnt:
repo: https://github.sec.samsung.net/RS6-RTOS/TizenRT
applied patch : https://github.com/Samsung/TizenRT/pull/4888

Result: Code stuck faced at Wi-Fi deinitialization step.
Wifi_conf.c >> Function wifi_off () >> rtlk_wlan_deinit() function execution is not finished. Logs are attached for reference.

I will share results here using github.com/Samsung/TizenRt repo and mentioned PR.

**Comment by vibhor-m** [comment]
The crash in deinit is fixed.

**Comment by sunghan-chang** [comment]
@vibhor-m Do you mean below is resolved?
```
vibhor-m commented 22 hours ago • 
Hello All,

My testing environment:
repo: https://github.sec.samsung.net/RS6-RTOS/TizenRT
applied patch : #4888

Result: Code stuck faced at Wi-Fi deinitialization step.
Wifi_conf.c >> Function wifi_off () >> rtlk_wlan_deinit() function execution is not finished.
I will share results here using github.com/Samsung/TizenRt repo and mentioned PR.
```

**Comment by vibhor-m** [comment]
> @vibhor-m Do you mean below is resolved?
> 
> ```
> vibhor-m commented 22 hours ago • 
> Hello All,
> 
> My testing environment:
> repo: https://github.sec.samsung.net/RS6-RTOS/TizenRT
> applied patch : #4888
> 
> Result: Code stuck faced at Wi-Fi deinitialization step.
> Wifi_conf.c >> Function wifi_off () >> rtlk_wlan_deinit() function execution is not finished.
> I will share results here using github.com/Samsung/TizenRt repo and mentioned PR.
> ```

This issue verification is still in progress.
Previously, there was a crash, crash is not occurring now.
Some more changes done in WM Test application, so we are in process of verification of this issue.

**Review by pillip8282 — APPROVED** [review]
lgtm

**Review by jeongchanKim — APPROVED** [review]
LGTM

**Review by an4967 — APPROVED** [review]
LGTM

**Review by sunghan-chang — APPROVED** [review]

**Pull request merged**